src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -528,10 +528,16 @@ describe('ReactDOMServerIntegration', () => {
         if (ReactDOMFeatureFlags.useFiber) {
           // with Fiber, there are just three separate text node children,
           // each of which is blank.
-          expect(e.childNodes.length).toBe(3);
-          expectTextNode(e.childNodes[0], '');
-          expectTextNode(e.childNodes[1], '');
-          expectTextNode(e.childNodes[2], '');
+          if (render === serverRender) {
```

Comment:
Once my other PR gets approved by reviewers I can start removing some of the differences. :)